### DISCUSSION TITLE
Adding more data

### DISCUSSION THREAD

**Discussion by oritzio** [faker-js/faker#1297] [Ideas]
Very cool repo.
I feel like you can add more data like sport teams of nba, nfl, soccer...

**Comment by Minozzzi** [comment]
I like this idea :D

**Comment by ST-DDT** [comment]
What would be the usecase for the data?

IMO these are highly specific data for a very limited field.
In most cases dummy data for "any team name" (e.g. city + animal) plus some random "fullname" players would work.

Maybe this could be useful for an optional module in the future.

**Reply by Minozzzi** [reply]
About soccer...
I think we could add the name of the major league teams to the locations.
For example:
The Premier League, major league in England, we can add team name as locality.
What do you think about?

**Reply by oritzio** [reply]
For example NFL teams:

[
    {
        "city": "Arizona",
        "name": "Cardinals",
        "abr": "ARI",
        "conf": "NFC",
        "div": "West"
    },
    {
        "city": "Atlanta",
        "name": "Falcons",
        "abr": "ATL",
        "conf": "NFC",
        "div": "South"
    },
    {
        "city": "Baltimore",
        "name": "Ravens",
        "abr": "BAL",
        "conf": "AFC",
        "div": "North"
    },
    {
        "city": "Buffalo",
        "name": "Bills",
        "abr": "BUF",
        "conf": "AFC",
        "div": "EAST"
    },

**Reply by oritzio** [reply]
I thought to create a package for that but since you already did a good job here I wanted to share that with you.
You can add many more

NBA:

  {
    "teamId": 1610612737,
    "abbreviation": "ATL",
    "teamName": "Atlanta Hawks",
    "simpleName": "Hawks",
    "location": "Atlanta"
  },
  {
    "teamId": 1610612738,
    "abbreviation": "BOS",
    "teamName": "Boston Celtics",
    "simpleName": "Celtics",
    "location": "Boston"
  },
  {
    "teamId": 1610612751,
    "abbreviation": "BKN",
    "teamName": "Brooklyn Nets",
    "simpleName": "Nets",
    "location": "Brooklyn"
  },
  {
    "teamId": 1610612766,
    "abbreviation": "CHA",
    "teamName": "Charlotte Hornets",
    "simpleName": "Hornets",
    "location": "Charlotte"
  },

**Reply by oritzio** [reply]
I created sports apps that I display a team vs team.
When I was developing that it would have been nice to generate it or maybe write a loop to add this to the db instead of manually.

**Comment by oritzio** [comment]
I feel like the category that you create in the API "Name" should be named as "Person" or "People"
because everything here is name, Person is the category of what you meant to create.

**Reply by ST-DDT** [reply]
That rename is planned for v8.
